doc/src/modules/physics/mechanics/symsystem.rst

```diff
@@ -0,0 +1,184 @@
+=====================================
+Symbolic Systems in Physics/Mechanics
+=====================================
+
+The `SymbolicSystem` class in physics/mechanics is a location for the pertinent
+of a dynamic system. In its most basic form it contains the equations of motion
+for the dynamic system, however, it can also contain information regarding the
+loads that the system is feeling, the bodies that the system is comprised of
+and any additional equations the user feels is important for the system. The
+goal of this class is to provide a unified output format for the equations of
+motion that simulation code can be designed around.
+
+SymbolicSystem Example Usage
+============================
+
+This code will go over the manual input of the equations of motion for the
+simple pendulum into `SymbolicSystem` using x and y coordinates instead of
+theta.
+
+The equations of motion are formed at
+http://nbviewer.jupyter.org/github/bmcage/odes/blob/master/docs/ipython/Planar%20Pendulum%20as%20DAE.ipynb ::
+
+    >>> from sympy import atan, symbols, Matrix
+    >>> from sympy.physics.mechanics import (dynamicsymbols, ReferenceFrame,
+    ...                                      Particle, Point)
+    >>> import sympy.physics.mechanics.system as system
+
+The first step will be to initialize all of the dynamic and constant symbols. ::
+
+    >>> x, y, u, v, lam = dynamicsymbols('x y u v lambda')
+    >>> m, l, g = symbols('m l g')
+
+Next step is to define the equations of motion in multiple forms:
+
+    [1] Explicit form where the kinematics and dynamics are combined
+        x' = F(x, t, r, p)
+
+    [2] Implicit form where the kinematics and dynamics are combined
+        M(x, p) x' = F(x, t, r, p)
+
+    [3] Implicit form where the kinematics and dynamics are separate
+        M(q, p) u' = F(q, u, t, r, p)
+        q' = G(q, u, t, r, p)
+
+where
+
+    x : states, i.e. [q, u]
+    t : time
+    r : specified (exogenous) inputs
+    p : constants
+    q : generalized coordinates
+    u : generalized speeds
+    M : mass matrix or generalized inertia matrix (implicit dynamical
+        equations or combined dynamics and kinematics)
+    F : right hand side (implicit dynamical equations or combined dynamics
+        and kinematics)
+    G : right hand side of the kinematical differential equations ::
```

Comment:
in restructured text you can prepend the lines with `|` to ensure each line is on a newline. I'd also move the `::` to its own line.